core/themes/basis/css/component/comment.css

```diff
@@ -163,3 +163,16 @@
 [dir="rtl"] .comment-footer {
   float: left;
 }
+
+a.comment-permalink  {
+  width: .7em;
+  height: .7em;
+  padding: .1em;
+  display: inline-block;
+  background-color: currentColor;
```

Comment:
What is `currentColor`?

Reply:
@BWPanda - I found where @indigoxela recommended this: 
https://github.com/backdrop/backdrop-issues/issues/4569#issuecomment-1120381705

Reply:
Ah, nice! Didn't know about that.